src/lib/draw-and-pick.js

```diff
@@ -206,11 +195,122 @@ export function pickLayers(gl, {
 }
 /* eslint-enable max-depth, max-statements */
 
+export function pickLayersByBoundingBox(gl, {
```

Comment:
> @ibgreen sorry, I'm having trouble finding this in the RFC. 

As mentioned, this is discussed in the comments in the RFC. But I will explain again:

Let's say you have 2 objects. But you are so zoomed out that they all render on a single pixel. You select a generous bounding box around them, expected the list of selected objects to include both objects. But only one of them will detected by this picking approach. 

Now repeat the thought experiment with more pixels and more objects. As long as there is a risk of one object overlapping with another, there is a risk that this picking method does not correctly identify all objects within the region, as it depends on a pixel representing that object actually being rendered into the framebuffer.